README.md

```diff
@@ -48,46 +48,99 @@ set.seed(1)
-| variance          | Variance                                                 | NA             | Integer, Numeric                                               | stats                                                                                                             |
+| Name              | label                                                    | Task Types     | Feature Types                                                  | Package                                                                                                           | Task Type |
+|:------------------|:---------------------------------------------------------|:---------------|:---------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------|:----------|
+| anova             | ANOVA F-Test                                             | Classif        | Integer, Numeric                                               | stats                                                                                                             | NA        |
+| auc               | Area Under the ROC Curve Score                           | Classif        | Integer, Numeric                                               | [mlr3measures](https://cran.r-project.org/package=mlr3measures)                                                   | NA        |
+| carscore          | Correlation-Adjusted coRrelation Score                   | Regr           | Numeric                                                        | [care](https://cran.r-project.org/package=care)                                                                   | NA        |
+| carsurvscore      | Correlation-Adjusted coRrelation Survival Score          | Surv           | Integer, Numeric                                               | [carSurv](https://cran.r-project.org/package=carSurv) , [mlr3proba](https://cran.r-project.org/package=mlr3proba) | NA        |
+| cmim              | Minimal Conditional Mutual Information Maximization      | Classif & Regr | Integer, Numeric, Factor, Ordered                              | [praznik](https://cran.r-project.org/package=praznik)                                                             | NA        |
+| correlation       | Correlation                                              | Regr           | Integer, Numeric                                               | stats                                                                                                             | NA        |
+| disr              | Double Input Symmetrical Relevance                       | Classif & Regr | Integer, Numeric, Factor, Ordered                              | [praznik](https://cran.r-project.org/package=praznik)                                                             | NA        |
```

Comment:
The README update sets a lot of values to NA. Is this caused by the PR or a general issue?

Reply:
I've updated the scripts to generate the table.